.travis.yml

```diff
@@ -55,10 +55,6 @@ matrix:
     compiler: ": #stack default"
     addons: {apt: {sources: [hvr-ghc],packages: [libgmp-dev,dnsutils,cabal-install-2.2]}}
 
-  - env: BUILD=stack ARGS="--resolver lts-6"
-    compiler: ": #stack 7.10.3"
-    addons: {apt: {sources: [hvr-ghc],packages: [libgmp-dev,dnsutils,cabal-install-1.22]}}
```

Comment:
Why are we dropping support?

Reply:
Dropping this resolver allowed me to bump the lower bounds of `directory` and remove a warning. How many versions of GHC is it customary to support for an application?

Reply:
@cydparser >How many versions of GHC is it customary to support for an application?

3 for libraries but tools often support more.

Reply:
> Dropping this resolver allowed me to bump the lower bounds of directory and remove a warning. 

Ok, but why are we bumping the lower bounds of `directory`? I'm not opposed to dropping support, but I'd like to ensure that it's worth it.